build.gradle.kts

```diff
@@ -108,7 +108,7 @@ subprojects {
 					version {
 						strictly("[2.12,3-alpha[")
 						// renovate: depName=com.fasterxml.jackson:jackson-bom
-						prefer("2.13.4")
+						prefer("2.13.4.20221013")
```

Comment:
This version doesn't exist for all of the jackson modules; as we're not impacted by this CVE I'm fine with waiting until `2.13.5`